.circleci/config.yml

```diff
@@ -88,17 +116,50 @@ jobs:
     docker:
       - image: 'node:6'
         user: node
-    steps: *unit_tests
+    steps:
```

Comment:
This feels like a lot of duplication.  Any way to reduce that?

Reply:
The [js-yaml](https://www.npmjs.com/package/js-yaml) module that I use for parsing and dumping does not do a good job with yaml references. I will try to make it look better.